cmd/crio/main.go

```diff
@@ -269,6 +269,18 @@ func main() {
 			return err
 		}
 
+		// Print the current cli flags
+		flagNames := c.FlagNames()
+		logrus.Infof("Cli Flags used:")
+		for _, flagName := range flagNames {
+			flagvalues := c.Value(flagName)
+			logrus.Infof("Flag: %v: %v \n", flagName, flagvalues)
```

Comment:
To keep the output format aligned with Kubernetes ecosystem style:

```suggestion
			logrus.Infof("FLAG: --%s=\"%v\"\n", flagName, flagvalues)
```